runtime/src/lib.rs

```diff
@@ -142,7 +144,7 @@ impl sdk::Runtime for Runtime {
         if option_env!("OASIS_UNSAFE_SKIP_KM_POLICY") == Some("1")
             || cfg!(feature = "debug-mock-sgx")
         {
-            return Some(TrustedSigners::default());
+            return Some(TrustedSigners::unsafe_mock());
```

Comment:
This won't work because `unsafe_mock` is only available under `debug-mock-sgx` feature. Use the variant mentioned by @CedarMist.